WTWSMS/common/dynasties/01_dynasties.txt

```diff
@@ -129,17 +129,6 @@
 10424381 = {
 	name="Elemunding"
 	culture = gepid
-	coat_of_arms = {
```

Comment:
Can this be the indended CoA?
![image](https://user-images.githubusercontent.com/29546927/52718734-6bb9ee00-2fa4-11e9-8307-b1c5ee59de1e.png)